changelog/62594.fixed

```diff
@@ -0,0 +1 @@
+win_lgpo: Display conflicting policy names when more than one policy is found
```

Comment:
the number 62594 seems to point to this issue: https://github.com/saltstack/salt/issues/62594

can you please change the filename to the issue that this PR Is fixing